### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by ran337287** [guoruoqian/DetNet_pytorch#5]
I set the optimizer as adam, and lr is 1e-4 due to the small training dataset, but loss may be nan, do you know what's  the problem?

**Comment by ran337287** [comment]
I checked my dataset, the coordinates range from 0 to size -1,  so I solved it.  Thank U！
